presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -886,6 +886,33 @@ public void testCreateUnpartitionedTableAsSelect()
         }
     }
 
+    @Test
+    public void testCreatePartitionedTableAsSelect()
+    {
+        {
+            Session session = buildSessionForTableWrite();
+            // Generate temporary table name for created partitioned table.
+            String partitionedOrdersTableName = generateRandomTableName();
+            // Clean up if temporary table already exists.
+            dropTableIfExists(partitionedOrdersTableName);
```

Comment:
This code and comment repeats in multiple places. It would be nice to extract it into a helper method for readability. Maybe this logic can be folded into generateRandomTableName?